modules/EG_Spectator_Mode/init.sqf

```diff
@@ -221,10 +221,23 @@ if (!isDedicated) then {
 				player setVariable ["FW_Spectating", true, true];
 				[true] call acre_api_fnc_setSpectator;
 				
+				//If babel is enabled, allowed spectator to hear all languages present in mission.
+				if (!isNil "FW_enable_babel" && FW_enable_babel) then {
```

Comment:
Add lazy evaluation (wrap `FW_enable_bable` in {}) to avoid game trying to retrieve value even if it's nil.